packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -163,7 +200,18 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
       return Math.max(this.props.min, Math.min(this.props.max, value));
     }
   }
-
+  private _getValueLabel(className: string): JSX.Element | null {
+    const { showValue, disabled, valueFormat, label } = this.props;
+    const value = valueFormat ? valueFormat(this.value!) : this.value;
+    if (showValue) {
+      return (
+        <Label className={className} disabled={disabled}>
+          {label ? `: ${value}` : value}
```

Comment:
Aren't we missing the `label` here, shouldn't this be:

```suggestion
          {label ? `${label}: ${value}` : value}
```

Reply:
wait no upon further inspection the label is already present before this method is even called